spark/v3.5/spark-runtime/LICENSE

```diff
@@ -355,14 +318,6 @@ License: http://www.apache.org/licenses/LICENSE-2.0
 
 --------------------------------------------------------------------------------
 
-This binary artifact contains Airlift Slice.
```

Comment:
I don't see airlift slide in the spark-runtime jar.

Reply:
<img width="510" alt="image" src="https://github.com/user-attachments/assets/dd4337d1-61b8-475d-a127-fa49da90b8ea" />
Similar to the above case, it's coming in from spark which we explicitly exclude